index.html

```diff
@@ -267,11 +267,11 @@
         <pre class="example highlight">
 &lt;!-- player.html --&gt;
 &lt;!-- The video element with custom controls that supports remote playback. --&gt;
-&lt;video src="https://example.org/media.ext" /&gt;
+&lt;video id="videoElement" src="https://example.org/media.ext" /&gt;
 &lt;button id="remotePlaybackBtn" style="display: none;"&gt;Play remotely&lt;/button&gt;
 &lt;script&gt;
   // The "Play remotely" button is visible if at least one remote playback device is available.
-  var remotePlaybackBtn = document.querySelector("video");
+  var remotePlaybackBtn = document.getElementById("remotePlaybackBtn");
```

Comment:
When I added this example, the reviewer suggested to use querySelector() vs. getElementById(). So I'm curious, what's the advantage of the latter over the former?
(I can see this being easier to copy/paste, for instance)